python/mxnet/base.py

```diff
@@ -483,8 +483,6 @@ def build_param_doc(arg_names, arg_types, arg_descs, remove_dup=True):
     for key, type_info, desc in zip(arg_names, arg_types, arg_descs):
         if key in param_keys and remove_dup:
             continue
-        if key == 'num_args':
```

Comment:
num_args is populated for variable length inputs without users having to worry about it. If we remove this the additional num_args field can confuse users.

Reply:
ok, then maybe I will write about providing 2 inputs for bilinear upsampling in the operator description.

Reply:
ok, then i'll just add a note in the description about providing 2 inputs for bilinear sampling